doc/languages-frameworks/rust.section.md

```diff
@@ -63,18 +64,7 @@ hash using `nix-hash --to-sri --type sha256 "<original sha256>"`.
 }
 ```
 
-Exception: If the application has cargo `git` dependencies, the `cargoHash`
-approach will not work by default. In this case, you can set `useFetchCargoVendor = true`
-to use an improved fetcher that supports handling `git` dependencies.
-
-```nix
-{
-  useFetchCargoVendor = true;
-  cargoHash = "sha256-RqPVFovDaD2rW31HyETJfQ0qVwFxoGEvqkIgag3H6KU=";
-}
-```
-
-If this method still does not work, you can resort to copying the `Cargo.lock` file into nixpkgs
+If this method does not work, you can resort to copying the `Cargo.lock` file into nixpkgs
```

Comment:
How about adding a nuance here, saying that while copying the `Cargo.lock` in `nixpkgs` works, it is not the best solution, and it should be used in last resort. In those case, an issue should be open upstream, asking the package author to include the file in the repo.

Reply:
That sounds like a good change but not in scope of changing fetchCargoTarball to fetchCargoVendor.